amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -148,33 +245,11 @@ impl Transform {
     }
 
     /// Set the rotation using Euler x, y, z.
-    pub fn set_rotation<D: Into<Deg<f32>>>(&mut self, x: D, y: D, z: D) -> &mut Self {
-        let rotation = Quaternion::from_angle_x(x.into()) * Quaternion::from_angle_y(y.into())
-            * Quaternion::from_angle_z(z.into());
-
-        self.rotation = rotation.into();
+    pub fn set_rotation<A: Into<Rad<f32>>>(&mut self, x: A, y: A, z: A) -> &mut Self {
+        self.rotation = Quaternion::from_angle_x(x) * Quaternion::from_angle_y(y)
+            * Quaternion::from_angle_z(z);
         self
     }
-
-    /// Calculate the view matrix from the given data.
-    pub fn to_view_matrix(&self, orientation: &Orientation) -> Matrix4<f32> {
```

Comment:
I think we should provide a `view_matrix()` function. It's the inverse of the normal matrix, but should be possible to calculate directly without doing the inverse explicitly.

Reply:
Ok

Reply:
You can defer this point to a followup PR.